dp/generate_climos.py

```diff
@@ -229,7 +243,7 @@ def curr_time_cdo_format():
 
     # Update generate_climos history attribute
     temporal_subset = update_generate_climos_history(
-        temporal_subset, t_start_generate_climos, 1
+        args, temporal_subset, t_start_generate_climos, 1
```

Comment:
Just passing in a ‘1‘ feels a little unintuitive. Perhaps setting the param as ‘position=1‘ would make it a bit easier to understand.